src/harness/unittests/commandLineParsing.ts

```diff
@@ -1,4 +1,4 @@
-﻿/// <reference path="..\harness.ts" />
+/// <reference path="..\harness.ts" />

```

Comment:
Exactly. `-<U+FEFF>/// <reference path="..\harness.ts" />` reported in git command line